docs/en_US/Release.md

```diff
@@ -1,5 +1,31 @@
 # ChangeLog
 
+## Release 0.9 - 7/1/2019
+
+### Major Features
+* General NAS programming interface
+    * Add `enas-mode` and `oneshot-mode` for NAS interface: [PR #1201](https://github.com/microsoft/nni/pull/1201#issue-291094510)
+* [Gaussian Process Tuner with Matern kernel](./GPTuner.md) 
+
```

Comment:
can we add how to use GPTuner here or in a doc? the GPTuner.md did not mention what's the tuner's name to be used in a configure.

Reply:
The usage of GPTuner is specified in docs/en_US/BuiltinTuner.md

Reply:
will be tracked in issue #1234